source/api/presentation/3.0/index.md

```diff
@@ -1093,6 +1097,25 @@ Any additional properties beyond those defined in this specification or the Web
 
 The JSON representation _MUST NOT_ include the `@graph` key at the top level. This key might be created when serializing directly from RDF data using the JSON-LD 1.0 compaction algorithm. Instead, JSON-LD framing and/or custom code should be used to ensure the structure of the document is as defined by this specification.
 
+### 4.7. Term Collisions between Contexts
+
+It must be noted that there are some common terms that are used in more than one JSON-LD context document. Every attempt has been made to minimize these collisions, but some are inevitable. In order to know which specification is in effect at any given point, the class of the resource that has the property is the governing factor. Thus properties on Annotation based resources use the context from the [Web Annotation Data Model][org-w3c-webanno], whereas properties on classes defined by this specification use the IIIF Presentation API context's definition.  
+
+The label property is defined by both and is in direct conflict:
+
+* `label`: The use of `label` in IIIF follows modern best practices for internationalization by allowing the language to be associated with the value using the language map construction [described above][prezi30-languages]. The Web Annotation Data Model uses it only for [Annotation Collections][prezi30-annocoll], and mandates that it must be a string.
+
+The following properties are defined by both, and the IIIF representation is more specific than the Web Annotation Data Model but are not in conflict, or are never used on the same resource:
+
+* `homepage`: In IIIF the home page of a resource must be represented as a JSON object, whereas in the Web Annotation Data Model it may also be a string.
```

Comment:
must or _MUST_ ?

Reply:
I wondered about that. I went with lower case as it's not /the/ normative requirement itself, it's just reflecting the requirement from the normative part in section 3. I'm fine to go either way.

Reply:
If we don't want to use normative MUST then say simply "is" to avoid confusion